src/client.ts

```diff
@@ -512,8 +515,32 @@ export class ServiceClient {
       headers: { Accept: 'application/json' },
     });
     await this.unwrap(response);
+    const json = await response.json();
 
-    return (await response.json()) as MapsListResponse;
+    return {
+      ...json,
+      data: json.data.map((map: unknown) => {
+        if (isMapMinimalReponse(map)) {
+          return {
+            map_id: map.id,
+            map_provider: '',
+            map_provider_url: '',
+            map_revision: '',
+            map_variant: null,
+            profile_name: '',
+            profile_url: '',
+            profile_version: '',
+            url: map.url,
+            published_at: new Date(),
+            published_by: '',
+            owner: '',
+            owner_url: '',
+          } as MapResponse;
```

Comment:
Suggestion: Is forced typing really needed?

```suggestion
          };
```

Reply:
sorry for this, I know I should not use force typing, fixed